polyply/src/load_library.py

```diff
@@ -17,14 +17,50 @@
 import vermouth
 from vermouth.gmx.rtp import read_rtp
 from vermouth.citation_parser import read_bib
+from vermouth.log_helpers import StyleAdapter, get_logger
 from polyply import DATA_PATH
 from .ff_parser_sub import read_ff
+from .build_file_parser import read_build_file
 from .polyply_parser import read_polyply
+from .topology import Topology
 
+LOGGER = StyleAdapter(get_logger(__name__))
 FORCE_FIELD_PARSERS = {'rtp': read_rtp, 'ff': read_ff, 'itp': read_polyply, 'bib': read_bib}
 
 
-def _resolve_lib_paths(lib_names, data_path):
+def check_extensions_bld(bld_files):
+    """
+    check the extensions of the
+    user provided build files
+
+    Parameters
+    ----------
+    bld_files: list[`pathlib.PosixPath`]
+        list of build files
+    """
+    for _file in bld_files:
```

Comment:
```suggestion
    for file_ in bld_files:
```
PEP8 says to post-fix underscores in these cases